CHANGES.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+
+Version 3.0.0
+-------------
+May Bär implemented multilayer support for the Server and Client. By this
```

Comment:
I would not put names in the Change Log.

Reply:
see 2.0 i would like to honor great changes